src/page/Coop/hook/useGetCurrentMenuType.ts

```diff
@@ -1,6 +1,9 @@
 import { Menus } from 'model/Coop';
+import dayjs from 'dayjs';
 
-const getCurrentMenuType = (): Menus => {
+export type IsOpen = '운영중' | '운영종료';
```

Comment:
운영 상태를 나누는 거니까 OperatingStatus는 어떤가요?
```suggestion
export type OperatingStatus = '운영중' | '운영종료';
```

Reply:
수정완료했습니다!